src/java/org/apache/cassandra/index/sai/disk/PostingListRangeIterator.java

```diff
@@ -146,9 +146,9 @@ private long getNextRowId() throws IOException
         long segmentRowId;
         if (needsSkipping)
         {
-            long targetRowID = primaryKeyMap.rowIdFromPrimaryKey(skipToToken);
+            long targetRowID = primaryKeyMap.firstRowIdFromPrimaryKey(skipToToken);
             // skipToToken is larger than max token in token file
-            if (targetRowID < 0)
+            if (targetRowID == Long.MAX_VALUE)
```

Comment:
This code isn't really reachable, AFAICT. I would like to merge this without adding coverage to it. Or, I can put the change into a larger PR some other time...